### PR TITLE
Updates from bedrock

### PR DESCRIPTION
**Pull request by MozmarRobot**
From file changes in https://github.com/mozilla/bedrock/commit/53a26385e8662c9fcb16abfe049293f33556e2a9

### CODE REVIEW

File: en/firefox/new/desktop.ftl [hunk 1]
```diff
@@ -15,6 +15,11 @@ firefox-desktop-download-meta-desc-v2 = Get { -brand-name-firefox } for { -brand
 firefox-desktop-download-og-title = Download the fastest { -brand-name-firefox } ever
 firefox-desktop-download-og-desc = Faster page loading, less memory usage and packed with features, the new { -brand-name-firefox } is here.
 firefox-desktop-download-firefox = { -brand-name-firefox-browser }
+
+# Variables:
+#   $update_url (url) - link to https://support.mozilla.org/kb/update-firefox-latest-release
+firefox-desktop-out-of-date = An even newer { -brand-name-firefox } is available. <a { $update_url }">Update to the latest version</a>
```
Comment (flodolo) [inline en/firefox/new/desktop.ftl:21]:
```suggestion
firefox-desktop-out-of-date = An even newer { -brand-name-firefox } is available. <a { $update_url }>Update to the latest version</a>
```

**Review by peiying2 — APPROVED** [review]

**Review by flodolo — CHANGES_REQUESTED** [review]

**Comment by peiying2** [comment]
@reemhamz there is an extra " in the `<a { $update_url }>Update to the latest version</a>` in your original PR. Make sure to change in the source. I should have not commit the change here.

File: en/firefox/new/desktop.ftl (hunk 1 above)
Comment (pmac) [inline en/firefox/new/desktop.ftl]:
```suggestion
firefox-desktop-out-of-date = An even newer { -brand-name-firefox } is available. <a { $update_url }>Update to the latest version</a>
```

Based on the code I think that quote mark is superfluous https://github.com/mozilla/bedrock/blob/53a26385e8662c9fcb16abfe049293f33556e2a9/bedrock/firefox/templates/firefox/new/desktop/download.html#L75

Comment (flodolo) [reply en/firefox/new/desktop.ftl]: It's happened at least a couple of times (causing a cascade of weird side effects on Smartling), I wonder if it's something that can be caught by linters.

Comment (pmac) [reply en/firefox/new/desktop.ftl]: Good question. Seems like it should be.

**Comment by reemhamz** [comment]
I'll get on the changes and post here when they're done -- thanks team

**Comment by reemhamz** [comment]
Updates have been merged in Bedrock 👍🏼

**Review by flodolo — APPROVED** [review]

**Pull request merged**